jujugui/static/gui/src/app/store/env/test-acl.js

```diff
@@ -0,0 +1,52 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+var juju = {components: {}}; // eslint-disable-line no-unused-vars
+
+chai.config.includeStack = true;
+chai.config.truncateThreshold = 0;
+
+describe('Deployment', function() {
+
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('acl', function() { done(); });
+  });
+
+  fit('can return initial readOnly state', function() {
+    var app = {
+      env: {
+        get: sinon.stub().withArgs('readOnly').returns(false)
+      }
+    };
+    var acl = new Y.juju.acl({app: app});
```

Comment:
Because you're passing `app` in as a param and not part of an options object this needs to be:
```JavaScript
var acl = new Y.juju.acl(app);
```